### ISSUE TITLE
Webpack role doesn't work

### ISSUE THREAD

**Issue by sephii** [liip/drifter#230]
The webpack role fails with an error when running `npm start`. I solved the problem by updating all the dependencies in `package.json` using `ncu`, then I was able to run `npm start`.

**Reproduction Steps:**

1. Create a blank project with the `webpack` role
2. Install kanbasu (`npm install --save-dev kanbasu && npx kanbasu assets/scss`)
3. Run `npm start`

**Expected behavior:**

The Webpack dev server is started.

**Observed behavior:**

```
$ npm start

> foobar@1.0.0 start /vagrant
> NODE_ENV=development webpack-dev-server --hot

/vagrant/node_modules/webpack-cli/bin/config-yargs.js:89
				describe: optionsSchema.definitions.output.properties.path.description,
				                                           ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (/vagrant/node_modules/webpack-cli/bin/config-yargs.js:89:48)
    at Object.<anonymous> (/vagrant/node_modules/webpack-dev-server/bin/webpack-dev-server.js:84:40)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
```

**Drifter version:** 1.2.0

**Comment by LeBenLeBen** [comment]
@sephii Which version of Node are you using?

**Comment by sephii** [comment]
```
npm@6.4.1
node@v8.12.0
```

**Comment by LeBenLeBen** [comment]
Fixed by #231
